modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -223,10 +220,9 @@ data:
 <3> The URL used to fetch a token with the specified `clientId` and `clientSecret`.
 <4> The OAuth 2.0 scopes for the authorization request. These scopes limit what data the tokens can access.
 <5> The OAuth 2.0 authorization request parameters required for the authorization server.
-====
 
-.Sample YAML for TLS client authentication
-====
+== Sample YAML for TLS client authentication
```

Comment:
[All headings in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings) take an anchor ID.